### PR TITLE
refactor: remove gray dividers from `Page` and `Sections`

### PR DESCRIPTION
**Pull request by goga-m**
## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

**Comment by faustbrian** [comment]
Task linked: [CU-egz2bj [Misc] Remove gray lines betweens page sections](https://app.clickup.com/t/egz2bj)

**Comment by ItsANameToo** [comment]
@goga-m conflict

**Comment by ItsANameToo** [comment]
@goga-m more conflicts :hide:

**Review by ItsANameToo — APPROVED** [review]

**Review by samharperpittam — APPROVED** [review]

File: src/domains/plugin/pages/PluginView/__snapshots__/PluginView.test.tsx.snap [hunk 1]
```diff
@@ -586,7 +586,7 @@ exports[`Plugin View should render plugin content with logo 1`] = `
       </div>
     </nav>
     <div
-      class="flex flex-col flex-1 mt-5"
+      class="flex flex-col flex-1"
     >
       <div
         class="bg-theme-background px-4 sm:px-6 lg:px-10 py-4 -mt-5"
```
Comment (luciorubeens) [inline src/domains/plugin/pages/PluginView/__snapshots__/PluginView.test.tsx.snap:592]: @goga-m can you remove this `-mt-5` from `PluginView`?

```suggestion
        class="bg-theme-background px-4 sm:px-6 lg:px-10 py-4"
```

Comment (goga-m) [reply src/domains/plugin/pages/PluginView/__snapshots__/PluginView.test.tsx.snap:592]: Sneak passed in conflict merge. Thanks, @luciorubeens

**Pull request merged**